scripts/Muon/GUI/Common/home_plot_widget/home_plot_widget_model.py

```diff
@@ -96,7 +96,7 @@ def remove_workpace_from_plot(self, workspace_name):
         self.plotted_workspaces = [item for item in self.plotted_workspaces if item != workspace_name]
         self.plotted_fit_workspaces = [item for item in self.plotted_fit_workspaces if item != workspace_name]
 
-    def _close_plot(self):
+    def close_plot(self):
```

Comment:
This isn't an issue introduced in this PR but seems sensible to address here anyway, can we clear self.plotted_workspaces_inverse_binning and self.plotted_fit_workspaces in close plot when we clear the list self.plotted_workspaces